app/models/user.rb

```diff
@@ -58,8 +58,8 @@ def as_json(_options = {})
       admin: admin }
     if (_options[:follows])
       json['follows'] = {
-        topics: following.where(followed_type: 'Topic').to_a.map(&:followed_id),
-        maps: following.where(followed_type: 'Map').to_a.map(&:followed_id)
+        topics: following.where(muted: false, followed_type: 'Topic').to_a.map(&:followed_id),
+        maps: following.where(muted: false, followed_type: 'Map').to_a.map(&:followed_id)
```

Comment:
we could consider creating a scope for this, maybe called `active` or something, so the code would become

    map: following.active.where(followed_type: 'Map').to_a.map(&:followed_id)

Reply:
right, makes sense